server/lib/job-queue/handlers/video-channels-sync.ts

```diff
@@ -0,0 +1,266 @@
+import { logger } from '@server/helpers/logger'
```

Comment:
Not sure whether the naming of this file is cool or not for this file.

I assumed we trigger an action (*sync*) for *video-channels*. So *video-channels* is here in plural. Are you happy with this name?

Reply:
Finally split video-channels-sync into 2 files:
- `server/lib/job-queue/handlers/video-channel-import.ts`
- `server/lib/schedulers/video-channel-sync-latest-scheduler.ts`